mmeval/metrics/multi_label.py

```diff
@@ -430,7 +430,7 @@ def _compute_metric(  # type: ignore
             pos_inds = (preds >= self.thr).long()
         else:
             # top-k labels will be predicted positive for any example
-            _, topk_indices = preds.topk(self.topk)
+            _, topk_indices = preds.topk(self.topk, dim=-1)
```

Comment:
The default value of `dim` should be `-1`.

Reply:
We will get an error if the dim is not specified since the upgrade of OneFlow:

OneFlow 0.8.0+cu112
![image](https://user-images.githubusercontent.com/32220263/210706900-64a5f34d-fe10-43b8-b477-d3edbcabc009.png)

OneFlow 0.8.1.dev20221226+cu112
![image](https://user-images.githubusercontent.com/32220263/210707578-dbaab524-cb64-44e4-9f42-d25eeed61aad.png)